components/articles/cards/ArticleCard.js

```diff
@@ -42,45 +42,80 @@ const ArticleCard = props => {
     linkProps: { route: ROUTES_NAMES.article, params: { slug } },
   };
 
-  if (type === 'video') {
-    return (
-      <CardWrapper {...wrapperProps} className="video" color={null}>
-        <VideoCard {...props} />
-      </CardWrapper>
-    );
-  }
-
-  if (collection) {
-    return (
-      <CardWrapper {...wrapperProps} className="collection">
-        <CollectionCard {...props} />
-      </CardWrapper>
-    );
-  }
-
   return (
-    <CardWrapper {...wrapperProps} className="article">
-      <div className="article__cover-wrapper">
-        <img className="article__cover article__cover--vertical" src={vertical} alt={title} />
-        <img className="article__cover article__cover--horizontal" src={horizontal} alt={title} />
+    <CardWrapper
+      {...wrapperProps}
```

Comment:
now it's redundant, just pass props explicitly

Reply:
what was the reason to have it on a first place?

Reply:
3 different `<CardWrapper` with some shared props